vnext/PropertySheets/CSharpNuGetReferences.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For both C++ and C# files:

Perhaps adhere to VS/MSBuilld naming conventions? Separate the language name with a dot. See:

```
Microsoft.Cpp.Default.props
Microsoft.Cpp.ManagedExtensions.props
Microsoft.Cpp.MSVC.Toolset.ARM64.props
Microsoft.Cpp.MSVC.Toolset.Common.props
Microsoft.Cpp.MSVC.Toolset.Win32.props
Microsoft.Cpp.MSVC.Toolset.x64.props
Microsoft.Cpp.MultiByteCharSupport.props
Microsoft.Cpp.Platform.props
Microsoft.Cpp.props
Microsoft.Cpp.UnitTest.props
Microsoft.Cpp.WindowsSDK.props
```

Reply:
Also see our own `vnext\PropertySheets\React.Cpp.props`.